smarts/sstudio/types.py

```diff
@@ -259,7 +259,7 @@ class MapSpec:
     """If specified, the default distance between pre-generated Lane Points (Waypoints)."""
     default_lane_width: Optional[float] = None
     """If specified, the default width (in meters) of lanes on this map."""
-    builder_fn: Optional[MapBuilder] = get_road_map
+    builder_fn: MapBuilder = get_road_map
```

Comment:
I'm not sure about this one since it's got a default value that isn't `None`.

Reply:
Since `builder_fn` has a default value which is not `None`, the `Optional[ ]` type was removed.

Reply:
oops, thanks.  I read that backwards originally.